DOCS-3003: Add prerequisite of physically assembling hardware to components and services (#3540)
viamrobotics/docs: docs/components/motor/dmc4000.md

The DMC-40x0 controller can drive a variety of motor types, but the built-in Viam implementation supports only stepper motors at this time.
You can drive other types of motors with Viam and the DMC-40x0 controller by creating a {{< glossary_tooltip term_id="modular-resource" text="modular resource" >}} to add support for it.

> Make sure to physically connect your motor to your machine's computer and turn it on.

Reviewer: If anyone is confused, the reason this one doesn't require a board to be configured is explained a few lines above.
```suggestion
Make sure to physically connect your motor to the DMC controller, and connect the controller to your machine's computer.
```

> Make sure to physically connect your motor to the DMC controller, and connect the controller to your machine's computer.

Reviewer 2: ```suggestion
Physically connect your motor to the DMC controller, and connect the controller to your machine's computer.
Power both on.
```
